README.md

```diff
@@ -169,19 +169,27 @@ Then open a browser and navigate to http://localhost:3000/
         ls slashdb
         auth.cfg  databases.cfg  license.key  nginx.conf  querydefs.cfg  slashdb.ini  taskdatadb.sqlite  users.cfg
 
-7.  Navigate to folder slashdb and run command:
+7.  Create folder for **SlashDB** logs.
+
+        mkdir slashdb-log
+
+8.  Create **SlashDB** docker container:
 
    Linux:
 
          docker run -d -p 8000:80 -v $(pwd):/etc/slashdb -v $(pwd):/var/log/slashdb slashdb/slashdb
```

Comment:
Please update the path to the slashdb-log folder for linux as well.

Reply:
Update made!